.github/workflows/test-zkvm-zkm.yml

```diff
@@ -0,0 +1,20 @@
+name: Test and clippy ZKM
```

Comment:
Maybe it's worth always use Ziren as the name everywhere? 
Since I think ZKM is the name of the team (similar to SP1 vs Succinct)